homeassistant/components/image_processing/facebox.py

```diff
@@ -104,16 +155,21 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     if DATA_FACEBOX not in hass.data:
         hass.data[DATA_FACEBOX] = []
 
-    entities = []
-    for camera in config[CONF_SOURCE]:
-        facebox = FaceClassifyEntity(
-            config[CONF_IP_ADDRESS],
-            config[CONF_PORT],
-            camera[CONF_ENTITY_ID],
-            camera.get(CONF_NAME))
-        entities.append(facebox)
-        hass.data[DATA_FACEBOX].append(facebox)
-    add_devices(entities)
+    ip_address = config[CONF_IP_ADDRESS]
+    port = config[CONF_PORT]
+    username = config.get(CONF_USERNAME)
+    password = config.get(CONF_PASSWORD)
+    url_health = "http://{}:{}/healthz".format(ip_address, port)
+    hostname = check_box_health(url_health, username, password)
+    if hostname is not None:
```

Comment:
Make it a guard clause by inverting the check and returning if the check is true.

Reply:
ok